js/src/Contexts/WidgetManager.js

```diff
@@ -511,23 +480,73 @@ export class WidgetManager extends Component {
         ...prevState,
         cachedForms: {
           ...prevState.cachedForms,
-          [itemBundle]: jsonData
+          [`${entityType}-${bundle}`]: jsonData
         }
       })))
       .catch(e => console.error(e));
   }
 
+  updateRowEntity(rowId, fieldName, newValues) {
```

Comment:
Can you add comments to each of these functions?

Reply:
Comments have been added